Unix/tools/auth_tests_setup.sh

```diff
@@ -37,34 +40,60 @@ do
 done
 
 #
-# if required env variables don't exist, then quit
+# Setup the NTLM auth file for client and server. They must be different files because 
+# if the differeing permissions, as the server file must be owned by root and the client by
```

Comment:
It shouldn't require any change in the product code. In  ValidateNtlmCredsFile the check is made to match the current uid/gid. If the server is running as root, then root, if it is running as omi_server_daemon, then that would be the ownership required.  

The file auth_tests_setup.sh would require changing one line (line 97) .

I don't beleive there would be any other changes.